src/Infrastructure/Client/ProductRelationAttributeClient.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\ExporterShopware6\Infrastructure\Client;
+
+use Ergonode\ExporterShopware6\Domain\Entity\Shopware6Channel;
+use Ergonode\ExporterShopware6\Domain\Repository\ProductRelationAttributeRepositoryInterface;
+use Ergonode\ExporterShopware6\Infrastructure\Connector\Action\ProductCrossSelling\GetAssignedProductsAction;
+use Ergonode\ExporterShopware6\Infrastructure\Connector\Action\ProductCrossSelling\GetCrossSellingAction;
+use Ergonode\ExporterShopware6\Infrastructure\Connector\Action\ProductCrossSelling\PatchCrossSellingAction;
+use Ergonode\ExporterShopware6\Infrastructure\Connector\Action\ProductCrossSelling\PostCrossSellingAction;
+use Ergonode\ExporterShopware6\Infrastructure\Connector\Shopware6Connector;
+use Ergonode\ExporterShopware6\Infrastructure\Exception\Shopware6InstanceOfException;
+use Ergonode\ExporterShopware6\Infrastructure\Model\AbstractProductCrossSelling;
+use Ergonode\ExporterShopware6\Infrastructure\Model\Basic\ProductCrossSelling;
+use Ergonode\ExporterShopware6\Infrastructure\Model\ProductCrossSelling\AbstractAssignedProduct;
+use Ergonode\ExporterShopware6\Infrastructure\Model\Shopware6Language;
+use Ergonode\SharedKernel\Domain\Aggregate\AttributeId;
+use Ergonode\SharedKernel\Domain\Aggregate\ProductId;
+
+class ProductRelationAttributeClient
+{
+    private Shopware6Connector $connector;
+
+    private ProductRelationAttributeRepositoryInterface $productRelationAttributeRepository;
+
+    public function __construct(
+        Shopware6Connector $connector,
+        ProductRelationAttributeRepositoryInterface $productRelationAttributeRepository
+    ) {
+        $this->connector = $connector;
+        $this->productRelationAttributeRepository = $productRelationAttributeRepository;
+    }
+
+    public function get(
+        Shopware6Channel $channel,
+        string $shopwareId,
+        ?Shopware6Language $shopware6Language = null
+    ): ?AbstractProductCrossSelling {
+
+        $action = new GetCrossSellingAction($shopwareId);
+        if ($shopware6Language) {
+            $action->addHeader('sw-language-id', $shopware6Language->getId());
```

Comment:
Wouldn't it be cleaner to move the appropriate header to action level? Sth like `addLanguageHeader`